.github/workflows/test.yml

```diff
@@ -0,0 +1,30 @@
+on: [push, pull_request]
+name: Test
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.14.x, 1.15.x]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Test
+      run: |
+        go test ./...
```

Comment:
We appear to have gained this vs Travis?

Reply:
Ah, I added this because I do for my repos, but wasn't thinking about it too much. I tend to prefer to run without and with the race detector, because it's cheap, and because it's clearer and faster to get "normal" failures without the race detector first, in my opinion. But I can remove this if either of you disagree.